### PR TITLE
new application menu design

### LINKED ISSUE #216: Collapsable App Sidebar
```
  <div :id="name">
    <div class="uk-offcanvas-bar uk-background-muted">
      <button class="uk-offcanvas-close" type="button" @click="onClose" uk-close></button>
      <ul class="uk-nav-default uk-nav-parent-icon uk-margin-medium-top" uk-nav="multiple: true">
        <li class="uk-parent uk-open">
            <router-link to="#" class="uk-text-uppercase">
              <oc-icon name="folder" /><span class="uk-text-uppercase">Files</span>
            </router-link>
            <ul class="uk-nav-sub">
                <li>
                  <router-link to="#">All files</router-link>
                </li>
                <li>
                  <router-link to="#">Favorites</router-link>
                </li>
                <li>
                  <router-link to="#">Shared by me</router-link>
                </li>
                <li>
                  <router-link to="#">Shared with me</router-link>
                </li>
                <li>
                  <router-link to="#">External storage</router-link>
                </li>
            </ul>
        </li>
        <li class="uk-nav-divider"></li>
        <li class="uk-parent">
            <router-link to="#">
              <oc-icon name="prototype" /><span class="uk-text-uppercase">Editor</span></router-link>
            <ul class="uk-nav-sub">
                <li>
                  <router-link to="#">Foo</router-link>
                </li>
                <li>
                  <router-link to="#">Bar</router-link>
                </li>
            </ul>
        </li>
        <li class="uk-nav-divider"></li>
        <li>
          <router-link to="#">
            <oc-icon name="image" /><span class="uk-text-uppercase">Gallery</span>
          </router-link>
        </li>
        <li class="uk-parent uk-margin-medium-top">
          <router-link to="#">
            <oc-icon name="account_circle" /><span class="uk-text-uppercase">Personal</span>
          </router-link>
          <ul class="uk-nav-sub">
            <li><router-link to="#">Profile</router-link></li>
            <li><router-link to="#">Storage</router-link></li>
            <li><router-link to="#">Security</router-link></li>
          </ul>
        </li>
        <li class="uk-nav-divider"></li>
        <li>
          <router-link to="#">
            <oc-icon name="application" /><span class="uk-text-uppercase">Administration</span>
          </router-link>
        </li>
        <li class="uk-nav-divider"></li>
        <li>
          <router-link to="#">
            <oc-icon name="info" /><span class="uk-text-uppercase">Help</span>
          </router-link>
        </li>
        <li class="uk-margin-medium-top">
          <router-link to="#">
            <oc-icon name="exit_to_app" /><span class="uk-text-uppercase">Logout</span>
          </router-link>
        </li>
      </ul>
    </div>
  </div>
```

Sass
```
.oc-main-menu {
    @extend .uk-nav-default;
    @extend .uk-nav-parent-icon;
    @extend .uk-margin-medium-top;

    .oc-icon {
        // 2px offset to align with line-height
        transform: translate(-50%, 3px);

        &, & > svg {
            width: $global-font-size + 1 !important;
            height: $global-font-size + 1 !important;
        }

        & > svg {
            fill : $global-color !important;
        }
    }
}
```

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Fixes #216 

Screenshot from 2019-04-11 14-52-10

### CODE REVIEW

**Comment by felixheidecke** [comment]
- [ ] Add dividers between groups (Applications, personal/admin and exit)
- [ ] Add margins between groups
- [ ] Drop icons in sub-menus

**Comment by DeepDiver1975** [comment]
Screenshot from 2019-04-11 14-44-30

File: src/elements/_OcSidebarNavItem.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <li :class="_class" class="oc-sidebar-nav-item">
     <component :is="_componentType" :to="target" v-on="!target ? { click: onClick } : {}">
       <span class="oc-sidebar-nav-item-wrapper">
```
[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply src/elements/_OcSidebarNavItem.vue]: hmmm - this kills the ```<``` on the right .... :confused:

[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply src/elements/_OcSidebarNavItem.vue]: no - something else killed it :angry:

**Comment by DeepDiver1975** [comment]
squashed my commits -> merge after ci is green

**Pull request merged**